api/src/opentrons/protocol_engine/state/protocol_run_data.py

```diff
@@ -0,0 +1,18 @@
+"""Protocol run data class for storing protocol engine state."""
```

Comment:
I'm never quite sure what to do with these module level docstrings, but I don't think we should use the word "store" here given its implications elsewhere in the codebase and the fact that this is a model for giving something a representation of the run state that is stored elsewhere

```suggestion
"""Public protocol run data models."""
```

Reply:
I like that docstring suggestion.

In addition, my hot take is that if a module contains only one class, like this one does, we should feel free to `# noqa` things so that we only have a docstring on the class *or* the module—not both.

If we have both, they'll certainly be redundant in a way that makes the code harder to understand and maintain.

But this is largely a personal pet peeve, and I also think it's reasonable to want to avoid littering our files with `# noqa` comments, so don't take me too seriously. :)